### PR TITLE
Add support for PHP 8.2, drop support for PHP 7.4

### PR DESCRIPTION
**Pull request by gsteel**
|    Q          |   A
|-------------- | ------
| New Feature   | yes

### CODE REVIEW

**Comment by gsteel** [comment]
This isn't gonna happen until `apcu` is available for PHP 8.2

**Comment by Ocramius** [comment]
Could we use a filesystem adapter?

**Comment by gsteel** [comment]
> Could we use a filesystem adapter?

No, FS adapter doesn't support static TTL :( It has to either redis or apcu I believe

**Comment by Ocramius** [comment]
Ah, that is unfortunate :| 

Redis will have the same issues then.

Can we somehow add static TTL support, perhaps?

**Comment by gsteel** [comment]
Shouldn't be too long before it's available in apt: https://github.com/krakjoe/apcu/releases/tag/v5.1.22

**Comment by Ocramius** [comment]
@gsteel with a rebase, this should be good to go, I think? :thinking: 

If not, we can install APCU manually, like we did in https://github.com/laminas/laminas-cache-storage-adapter-apcu/pull/45

File: .laminas-ci.json [hunk 1]
```diff
@@ -4,5 +4,8 @@
     ],
     "ini": [
         "apc.enable_cli=1"
-    ]
+    ],
+    "ignore_php_platform_requirements": {
+        "8.2": true
```
Comment (Ocramius) [inline .laminas-ci.json]: This can now be dropped

**Review by Ocramius — CHANGES_REQUESTED** [review]

File: .laminas-ci.json (hunk 1 above)
Comment (gsteel) [reply .laminas-ci.json]: Yes - I'll do that now 👍

**Review by Ocramius — APPROVED** [review]
Thanks @gsteel!

**Pull request merged**